### DISCUSSION TITLE
PSM3 SUJ: manually lowering the arm

### DISCUSSION THREAD

**Discussion by Sun-r1s3** [jhu-dvrk/sawIntuitiveResearchKit#190] [General]
Dear all,
In the clinical da-Vinci setup, the PSM3 is raised/lowered with a motor. In clinical mode, however, our system does not allow us to lower it, leaving it at a height that hinders the positioning of the other arms (in particular the PSM2). I know that there is a dvrk controller for SUJs that allows you to control this motorized joint, but in our setup it is not available. Is there an alternative solution even just to lower it once and then leave it in that position?

**Comment by pkazanzides** [comment]
I was hoping someone else could answer, since this has come up before.  The solution is to attach a power supply to the motor lines on the connector (Motor+ is pins A1,A3,A5 and Motor- is pins A2,A4,A6).  If you use an adjustable power supply, you can even control the speed.  I think a 24V supply would be enough.

If you want to be careful, you can measure the resistance between the Motor+ and Motor- pins.  For a motor of this size, I expect it would be somewhere between 1 Ohm and 10 Ohms.
